src/profile-logic/process-profile.js

```diff
@@ -976,7 +976,7 @@ export function unserializeProfileOfArbitraryFormat(
 }
 
 export class ProfileProcessor {
-  processProfile(geckoProfile: GeckoProfile) {
+  processProfile(geckoProfile: GeckoProfile): Promise<*> {
```

Comment:
Now, Flow wants that all generics are specified. And for exported functions we need to be stricter too. So it couldn't understand what value this Promise holds without this.

See https://github.com/facebook/flow/issues/4923 for a Flow bug about this -- I think Flow should be able to infer this by itself but well, this is an easy workaround.